src/frontend/src/optimizer/plan_node/stream_sink.rs

```diff
@@ -427,6 +429,15 @@ impl StreamSink {
             SinkLogStoreType::InMemoryLogStore
         };
 
+        // sink into table should have logstore for sink_decouple
+        println!("any target table: {:?}", target_table.is_some());
```

Comment:
Change `println` to `trace` or `debug`?

Reply:
https://github.com/risingwavelabs/risingwave/pull/21784/commits/84352c69639a710b636a3382540fe687a8981885